ci/generate-and-upload-build-steps.sh

```diff
@@ -35,10 +55,21 @@ generate_build_configuration_steps () {
         else
             echo "Building for all supported configurations. Generating the appropriate steps..."
 
+            export MACHINE_TYPE_ENVVAR="single-high-cpu" # run the weekly with smaller nodes, since this is not time-critical
+
             # Editor builds (Test and Shipping build states do not exist for the Editor build target)
             for BUILD_STATE in "DebugGame" "Development"; do
                 upload_build_configuration_step "${ENGINE_COMMIT_HASH}" "Win64" "Editor" "${BUILD_STATE}"
             done
+
+            # Generate all possible builds for non-Editor build targets
+            for BUILD_PLATFORM in "Win64" "Linux"; do
```

Comment:
next step: add mac into the weekly build ;) 
if we are doing non-editor build targets: would it make sense to add android (and iOS) as well?

Reply:
Hm, yes, we probably can. Currently mac would still be built, but just as a nightly, maybe it would be nice to have everything in one place.

Reply:
Although iOS would require the mac agents, and it is quite a lot of builds.

Reply:
if it is a weekly build, it might be fine. How long do they take to run?

Reply:
https://buildkite.com/improbable/unrealgdk-premerge/builds/11483

Reply:
oh yeah, they are a bit long 😓 
if we had to choose one, I would prefer getting iOS in as we only really support Mac for editor purposes anyway. but we can create a ticket and do that as a follow up PR as well